src/hsmm_mvpy/models.py

```diff
@@ -534,7 +610,7 @@ def compute_max_events(self):
         '''
         Compute the maximum possible number of events given event width and mean or minimum reaction time
         '''
-        return int(np.min(self.durations)//(self.event_width_samples))
+        return int(np.rint(np.min(self.durations)//(self.location)))-2
```

Comment:
-1? Only the first stage doesn't have a location

Reply:
Indeed